README.md

```diff
@@ -1,3 +1,85 @@
-# Triton-optimized Couchbase
+# Autopilot pattern for Couchbase
 
-Read [the blog post](https://www.joyent.com/blog/couchbase-in-docker-containers) and the [Docker Compose yaml](https://github.com/misterbisson/clustered-couchbase-in-containers) to better understand how to use this image.
\ No newline at end of file
+This repo is a demonstration of the [autopilot pattern](http://autopilotpattern.io/) as applied to [Couchbase](http://www.couchbase.com/). Couchbase's built-in cluster awareness and automatic management of data, including sharding and cross-datacenter replication make it ideal for deployment as a part of an entire stack using the autopilot pattern.
```

Comment:
I was thinking I'd add an update note at the top directing people to see it in context in the new modern application blueprint demonstrating the telemetry features. 

...and to the updated repo readme for couchbase, which probably has all the detail we need for the standalone db.